### DISCUSSION TITLE
v. 0.9.0 DataPlane not found

### DISCUSSION THREAD

**Discussion by JorgeBarto** [eclipse-edc/Connector#4481] [General]
Hi everyone,
I'm using the newest version, 0.9.0 and I have even updated database tables included because I'm using postgreSQL persistence.
Creating asset, policies, contract... everything look fine but, the problem is while requesting a Pull Transfer, I'm getting the follow error:

SEVERE 2024-09-18T11:00:29.762281432 TransferProcess: ID c3de7db2-e5cf-48fe-b7c1-3ac3527ba9a6. Fatal error while Initiate data flow. Error details: **DataPlane not found**

Besides, without using postgreSQL persistence, Pull transfer goes ok.

Wthat I might be forgetting??
Thanks a lot!

**Comment by jimmarino** [comment]
Can you please post these types of questions on the Discord channel? Discussions are for specific code-related questions. Thanks.

**Reply by JorgeBarto** [reply]
@jimmarino sorry about that, I do not know what you mean by "Discord channel".
I just see these topics in View all discussions:
Announcements, **General**, Ideas, Kind words, Planning, Polls, Q&A, Show and tell.

**Reply by jimmarino** [reply]
https://discord.gg/n4sD9qtjMQ
